dbt/tests/generic/test_sequential_values.sql

```diff
@@ -0,0 +1,53 @@
+-- Override dbt_utils.sequential_values to support `additional_select_columns` parameter
```

Comment:
See the original test definition for comparison: https://github.com/dbt-labs/dbt-utils/blob/23da1f424398696e25656dc6f89066b9a5d13794/macros/generic_tests/sequential_values.sql#L1-L42